message_subscribe.module

```diff
@@ -130,7 +130,16 @@ function message_subscribe_send_message($entity_type, $entity, Message $message,
   if ($subscribe_options['uids']) {
     // We got a list of user IDs directly from the implementing module,
     // However we need to adhere to the range.
-    $uids = $subscribe_options['range'] ? array_slice($subscribe_options['uids'], 0, $subscribe_options['range'], TRUE) : $subscribe_options['uids'];
+    // If 'last uid' is provided, we need to start from next 'uid' key in array.
+    if (!empty($subscribe_options['last uid'])) {
+      $index = array_search($subscribe_options['last uid'], array_keys($subscribe_options['uids'])) + 1;
```

Comment:
@capuleto 
Can you please add a check that determine if the `last uid` is in the `uids` array?
If it's not then just use same fallback (set `$index = 0;`) and add this scenario to the docs.

If you can add a test for that scenario as well it can be awesome :)

@amitaibu 
This should cover it.